common.gypi

```diff
@@ -105,7 +105,7 @@
       [ 'OS in "linux freebsd openbsd solaris android"', {
         'cflags': [ '-Wall', '-Wextra', '-Wno-unused-parameter', '-Wno-comment' ],
         'cflags_cc!': [ '-fno-rtti', '-fno-exceptions' ],
-        'cflags_cc': [ '-std=c++11' ],
+        'cflags_cc': [ '-std=c++14' ],
```

Comment:
NodeJS requires C++14 onwards, the gyp (a build system used by Node) is used by Protagonist, the Drafter NodeJS bindings. Change needed for NodeJS 16 compatibility.